package.json

```diff
@@ -91,7 +91,7 @@
     "react-native-device-info": "^13.2.0",
     "react-native-exit-app": "^2.0.0",
     "react-native-fast-image": "^8.6.3",
-    "react-native-gesture-handler": "^2.21.2",
+    "react-native-gesture-handler": "^2.23.1",
```

Comment:
Could you also update the version in the `peerDependencies` of `packages/@divvi/mobile`?

Reply:
Fixed in 1cd298995999989bedc85c42e60281e1db08bd57.